### PR TITLE
Control-Tower Changes - Migrating from AWS API polling integrations to Metrics Streaming

### PR DESCRIPTION
**Pull request by SuruchiSaboo**
Control-Tower Changes - Migrating from AWS API polling integrations to Metrics Streaming

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/partner_editable/_settings.adoc [hunk 1]
```diff
@@ -1,18 +1,18 @@
 // Change the following attributes.
 :quickstart-project-name: quickstart-ct-newrelic-one
-:partner-product-name: New Relic AWS Control Tower integration
+:partner-product-name: New Relic AWS Control Tower Integration
```
Comment (DelfinGala) [inline docs/partner_editable/_settings.adoc:3]:
```suggestion
:partner-product-name: New Relic AWS Control Tower integration
```

File: docs/partner_editable/additional_info.adoc [hunk 2]
```diff
@@ -1,21 +1,21 @@
 == Test the deployment
 
-When you enroll a new AWS Control Tower–managed account, the deployment sets up the IAM role for {partner-product-short-name}. You can then see the new account show up in the New Relic UI.
+When you enroll a new AWS Control Tower–managed account, the deployment sets up the IAM role and Metrics Streaming Resources for {partner-product-short-name}. You can then see the new account show up in the New Relic UI.
```
Comment (DelfinGala) [inline docs/partner_editable/additional_info.adoc:3]: Looking at the catalog, "Metrics Streaming Resources" isn't listed. Should this be [monitoring](https://docs.aws.amazon.com/controltower/latest/userguide/monitoring-overview.html)?

File: docs/partner_editable/additional_info.adoc [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+WARNING: Deleting the {partner-product-short-name} Quick Start stack and StackSets also deletes the New Relic IAM roles deployed in your AWS accounts, which removes cross-account trust with New Relic. This Quick Start does not, however, remove account links to New Relic. To remove an account from New Relic, see the subsequent section *Uninstall the new Relic integration*.
 
 === Remove the AWS CloudFormation stack
 
 The time to complete this step depends on how many AWS accounts are included in your {partner-product-name} deployment. If deleting the AWS CloudFormation stack times out, it's safe to retry this step.
 
 . Sign in to the https://console.aws.amazon.com/console/home[AWS CloudFormation console^] in your Control Tower management account.
-. To remove all of the deployed AWS resources, delete the {partner-product-name} stack. This also deletes all stack set instances, which include any IAM roles deployed for enrolled AWS accounts. 
+. To remove all of the deployed AWS resources, delete the {partner-product-name} stack. This also deletes all stack set instances, which include any IAM roles and Metrics Streaming Resources deployed for enrolled AWS accounts.
```
Comment (DelfinGala) [inline docs/partner_editable/additional_info.adoc:18]: Looking at the catalog, "Metrics Streaming Resources" isn't listed. Should this be [monitoring](https://docs.aws.amazon.com/controltower/latest/userguide/monitoring-overview.html)?

File: docs/partner_editable/architecture.adoc [hunk 4]
```diff
@@ -12,4 +12,4 @@ image::../images/newrelic-one-ct-integration-arch.png[New Relic AWS Control Towe
 * A dead letter queue collects messages that the New Relic register Lambda function fails to process.
 * A NerdGraph API key secret is retrieved from AWS Secrets Manager (not shown). The New Relic register Lambda function calls the New Relic NerdGraph mutation endpoint to link your AWS account with your New Relic account.
 * A New Relic onboarding Lambda function invokes the stack SNS topic and provides input to the New Relic StackSet, which includes a cross-account IAM role.
-* The New Relic StackSet creates a New Relic IAM cross-account role.
\ No newline at end of file
+* The New Relic StackSet creates a New Relic IAM cross-account role, CloudWatch Metric Stream, FireHose Delivery Stream and their respective Roles to stream metrics to NewRelic Platform.
```
Comment (DelfinGala) [inline docs/partner_editable/architecture.adoc:15]:
```suggestion
* The New Relic stack set creates an IAM cross-account role, an AWS CloudWatch metric stream, a Kinesis Data Firehose delivery stream, and respective roles to deliver metrics to the New Relic platform.
```

File: docs/partner_editable/faq_troubleshooting.adoc [hunk 5]
```diff
@@ -47,3 +47,7 @@ New Relic offers several https://developer.newrelic.com/try-our-apis/[APIs^] for
 *Q.* I included a list of AWS accounts during the deployment, but none of the accounts show up in New Relic's UI.
 
 *A.* Ensure that your list includes only AWS account IDs that are managed by AWS Control Tower and that the account IDs are separated by commas. Also ensure that you deployed the stack in the same Region as your AWS Control Tower management account and that the New Relic NerdGraph user API key is correct. Lastly, inspect the Amazon CloudWatch logs for errors regarding the AWS Lambda function `NewRelicOnboardingFunction`.
+
+*Q.* How are Metrics from AWS Account sent to New Relic Platform.
```
Comment (DelfinGala) [inline docs/partner_editable/faq_troubleshooting.adoc:51]:
```suggestion
*Q.* How are metrics delivered to the New Relic platform?
```

File: docs/partner_editable/pre-reqs.adoc [hunk 6]
```diff
@@ -31,3 +31,7 @@ In case you've already set up your landing zone, and want to monitor existing AW
 * Log in to an AWS Control Tower management account.
 * For a list of managed accounts, navigate to https://console.aws.amazon.com/controltower/home/accounts?[AWS Control Tower^].
 * Add your AWS account IDs.
+. {partner-company-name} License Key
```
Comment (DelfinGala) [inline docs/partner_editable/pre-reqs.adoc:34]:
```suggestion
. {partner-company-name} license key
```

File: docs/partner_editable/pre-reqs.adoc [hunk 7]
```diff
@@ -31,3 +31,7 @@ In case you've already set up your landing zone, and want to monitor existing AW
...
++
+The license key associated with the New Relic account you wish to export metrics to is required for sending data. License key can be generated from New Relic UI - https://one.newrelic.com/admin-portal/api-keys/home
```
Comment (DelfinGala) [inline docs/partner_editable/pre-reqs.adoc:36]: To export metrics to your New Relic account, you must have a license key. To generate a new key, refer to https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/[New Relic API keys^].

File: docs/partner_editable/product_description.adoc [hunk 8]
```diff
@@ -1,6 +1,6 @@
...
+{partner-product-short-name} requires that you grant read permission to operational telemetry data from your AWS account. To do this, use an AWS Identity and Access Management (IAM) role that uses https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html[IAM cross-account access^]. {partner-product-short-name} uses Amazon CloudWatch Metric Stream and Firehose Delivery Stream to collect and stream telemetry data for your monitored AWS services to New Relic. For more information, see https://docs.newrelic.com/docs/integrations/amazon-integrations/get-started/integrations-managed-policies[Integrations and managed policies^].
```
Comment (DelfinGala) [inline docs/partner_editable/product_description.adoc:4]:
```suggestion
{partner-product-short-name} requires that you grant read permission to operational telemetry data from your AWS account. To do this, use an AWS Identity and Access Management (IAM) role that uses https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html[IAM cross-account access^].  To collect and stream your monitored telemetry data to New Relic, {partner-product-short-name} uses an Amazon CloudWatch metric stream and a Kinesis Firehose delivery stream. For more information, refer to https://docs.newrelic.com/docs/integrations/amazon-integrations/get-started/integrations-managed-policies[Integrations and managed policies^]. 
```

**Review by DelfinGala — CHANGES_REQUESTED** [review]
See my inline comments and let me know if you have any questions.

File: docs/partner_editable/additional_info.adoc (hunk 2 above)
Comment (anshrma) [reply docs/partner_editable/additional_info.adoc:3]: No. This solution is not monitoring control tower. This solution is bootstrapping new accounts created by CT to ship metrics data using [CW metrics streams](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Metric-Streams.html)

File: docs/partner_editable/additional_info.adoc (hunk 3 above)
Comment (anshrma) [reply docs/partner_editable/additional_info.adoc:18]: No. This solution is not monitoring control tower. This solution is bootstrapping new accounts created by CT to ship metrics data using [CW metrics streams](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Metric-Streams.html)

**Comment by SuruchiSaboo** [comment]
@DelfinGala, I've made all required changes, could you please review the PR again.

**Pull request merged**